pan3d/__init__.py

```diff
@@ -1,15 +1,16 @@
 import logging
 from .dataset_builder import DatasetBuilder
 
+
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)
 
 __version__ = "0.8.9"
 
 try:
     from .dataset_viewer import DatasetViewer
-
-    __all__ = [DatasetBuilder, DatasetViewer]
+    from .explorers.slice_explorer import SliceExplorer
```

Comment:
I don't think it is a good idea to put it at the top package especially since trame is supposed to be optional for pan3d in case you only care about the dataset_builder.

Reply:
I see the try catch now... Never mind.